Acumatica.RESTClient/BaseApi/ODataAPI.cs

```diff
@@ -1,23 +1,58 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.Text;
-
 using Acumatica.RESTClient.Client;
-
 using RestSharp;
+using RestSharp.Authenticators;
 
 namespace Acumatica.RESTClient.Api
 {
+   
     public class ODataAPI : BaseApi
     {
+        RestClient client;
+        string basePath;
+
         public ODataAPI(Configuration configuration) : base(configuration)
         {
+            basePath = configuration.BasePath;
         }
+        /*
         public string Get(string path, string select = null, string filter = null, string expand = null, string custom = null, int? skip = null, int? top = null)
         {
             ApiResponse<string> localVarResponse = GetWithHttpInfo(path, select, filter, expand, custom, skip, top);
             return localVarResponse.Data;
         }
+        */
+        public string Get(string version, string resource = null, Dictionary<string, string> parameters = null, string username = null, string password = null)
+        {
+            if(username != null && password != null)
+            {
+                //get for login
+                basePath += "/" + version;
+                client = new RestClient(basePath);
```

Comment:
For Odata request with multitenant environment the url is not correct here. We need companyID part in the url